docs/source/tutorial_notebooks/getting_started.ipynb

```diff
@@ -31,15 +23,7 @@
     "\n",
     "### Before we begin\n",
     "\n",
-    "First off, make sure that you have the `dataregistry` package installed and available to you at NERSC (installation instructions [here](http://lsstdesc.org/dataregistry/installation.html)). This is generally simplest within a *Conda* or Python virtual environment (note that the `dataregistry` is not yet part of the `desc-python` environment). \n",
-    "\n",
-    "Also, make sure you have completed the one-time-setup for DREGS (more details [here](http://lsstdesc.org/dataregistry/installation.html)), namely:\n",
-    "\n",
-    "- You have added the database connection information in your `~/.config_reg_access` file\n",
-    "- You have added your database authentication information in your `~/.pgpass` file\n",
-    "\n",
-    "Finally, for this notebook to work for you interactively, you will need to be operating within a kernel that has the `dataregistry` package installed (see [here](https://docs.nersc.gov/services/jupyter/how-to-guides/) how to register your own Python environments as Jupyter kernels).\n",
-    "\n",
+    "If you haven't done so already, check out the [getting setup](file:///home/mcalpine/Documents/dataregistry/dataregistry/docs/build/html/tutorial_setup.html) page from the docs if you want to run this tutorial interactively.\n",
```

Comment:
I don't think you want a reference to what looks like a local file on your machine.